### ISSUE TITLE
`@export` open scene button not working.

### ISSUE THREAD

**Issue by hannan-ahmad** [godotengine/godot#100649]
### Tested versions

- Reproducible in v4.4.dev7.official [46c8f8c5c]
- Not reproducible in v4.3.stable.official [77dcf97d8]

### System information

Godot v4.4.dev7 - Linux Mint 22 (Wilma) on X11 - X11 display driver, Single-window, 1 monitor - OpenGL 3 (Compatibility) - Mesa Intel(R) HD Graphics 520 (SKL GT2) - Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz (4 threads)

### Issue description

The OpenScene Button does not appear to work when using `@export` annonations when you create an `@export` variable of type `PackedScene`, or `Array[PackedScene]`.

### Steps to reproduce

```gdscript
extends Node

@export var item_scenes: PackedScene
```

Now, you will see in your inspector like this:

![Image](https://github.com/user-attachments/assets/018d4c26-a12c-447a-b321-8ddae840fd23)

### Minimal reproduction project (MRP)

```gdscript
extends Node

@export var item_scenes: PackedScene
```

**Comment by mdelorme** [comment]
The problem seems to be coming from commit ([7f09804](https://github.com/godotengine/godot/commit/7f098041546eb877ce2b28a4a127245995725fe5)) where a second parameter was added to `open_request` (see [here](https://github.com/godotengine/godot/blob/7f098041546eb877ce2b28a4a127245995725fe5/editor/editor_node.h#L771)). 

This parameter has a default value, but since the [call](https://github.com/godotengine/godot/blob/master/editor/plugins/packed_scene_editor_plugin.cpp#L39) in the packed scene is deferred, the templating requires this second parameter passed explicitly to `call_deferred`. Not clear why the code would not give a compilation error though.

I can try to propose a fix.

**Comment by KoBeWi** [comment]
Fixed by #100654
